src/redux/reducers/rootReducer.js

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+import user from './user';
+import flashMessages from './flashMessages';
+
+export default combineReducers({
+    user,
```

Comment:
I think you should put the reducers in key and value to easily identify them by their keys when you are using  in the mapStateToProps in your react component